damien/models/evaluation.py

```diff
@@ -104,7 +104,7 @@ def __init__(
         self.term_id = term_id
         self.course_number = course_number
         self.department_id = department_id
-        self.instructor_uid = instructor_uid
+        self.instructor_uid = instructor_uid if instructor_uid != 'None' else None
```

Comment:
@cesarvh - Just curious, is 'None' going in the db when it should be NULL?

Reply:
@johncrossman Yes, that exactly was the bug. It has to do with "None" being used in placeholder id strings for evaluation rows without an instructor.